Website/index.js

```diff
@@ -0,0 +1,471 @@
+            $(v).hide();
+        });
+        $("#overview").show();
+        $("#start").attr("onchange", "update()")
+        update();
+    } else {
+        $("#overview").hide();
+        $("#start").attr("onchange", "show_personalised()")
+        show_personalised();
+    }
+}
+
+$.each(["#weekend", "#date", "overview"], function (i, v) {
+    $(v).hide();
+});
+
+var jsonData;
+var flagData;
+var nameData;
+
+$.getJSON("Data/data.json?time=" + new Date().getTime(),
+    function (data) {
+        var dt = new Date();
+        const monthFixer = ["01", "02", "03", "04", "05", "06", "07", "08", "09", "10", "11", "12"];
+
+        date = String(dt.getFullYear()) + "-" + String(monthFixer[dt.getMonth()]) + "-" + String(dt.getDate());
+        $("#start").attr("min", Object.keys(data["dates"])[0]);
+        $("#start").attr("max", Object.keys(data["dates"])[Object.keys(data["dates"]).length - 1]);
+
+        function formatDate(date) {
+        var d = new Date(date),
+            month = '' + (d.getMonth() + 1),
+            day = '' + d.getDate(),
+            year = d.getFullYear();
+
+        if (month.length < 2) 
+            month = '0' + month;
+        if (day.length < 2) 
+            day = '0' + day;
+
+        return [year, month, day].join('-');
+        }
+        $("#start").val(formatDate(dt));
+
+        jsonData = data;
+        getFlagsData();
+
+    }
+);
+
+function getFlagsData() {
+
+    $.getJSON("Data/flag.json?time=" + new Date().getTime(),
+        function (data) {
+            flagData = data;
+            getNameData();
+        }
+    );
+}
+
+function getNameData() {
+
+    $.getJSON("Data/names.json?time=" + new Date().getTime(),
+        function (data) {
+            nameData = data;
+
+            $.each(Object.keys(nameData), function (i, v) {
+                
+            if (localStorage.getItem("name") != null && localStorage.getItem("name") == v) {
+                $('option:selected', name_og).remove();
+                var a = '<option selected value="' + String(v) + '">' + String(v) + '</option>'
+                $(a).appendTo("#name");
+            } else {
+                var a = '<option value="' + String(v) + '">' + String(v) + '</option>'
+                $(a).appendTo("#name");
+            }
+                
+            });
+
+            name_checker();
+        }
+    );
+}
+
+var blink = false;
+
+function recess_time(period) {
+    var icon = '<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-record-circle mx-1 icon-blink" style="fill: #ff0000" viewBox="0 0 16 16"><path d="M8 15A7 7 0 1 1 8 1a7 7 0 0 1 0 14zm0 1A8 8 0 1 0 8 0a8 8 0 0 0 0 16z" /><path d="M11 8a3 3 0 1 1-6 0 3 3 0 0 1 6 0z" /></svg>'
+
+    function blink_text() {
+        blink = true
+        $('.icon-blink').each(function () {
+            var elem = $(this);
+            setInterval(function () {
+                if (elem.css('visibility') == 'hidden') {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Davianlols6/SC-Duty-List/pullRequest?prid=8386818)
```suggestion
                if (elem.css("visibility") == 'hidden') {
```